LEAF_Request_Portal/templates/subindicators.tpl

```diff
@@ -475,14 +475,18 @@
             formValidator["id<!--{$indicator.indicatorID}-->"] = {
                 setValidator: function() {
                     let value = $('#<!--{$indicator.indicatorID|strip_tags}-->').val().trim();
+                    if (value === ''){
```

Comment:
Please get rid of all the "returns".  LINT screams at that.  The accepted method is to declare a return variable at the top of the function (ex $isNumeric = false;) and then fill/modify it as the code goes.  This keeps it to a single return and also helps maintain the return _type_.  Very important when we start tightening up our function calls.  ;-)